src/Repository/SuiviRepository.php

```diff
@@ -115,17 +117,18 @@ public function findSignalementNoSuiviSince(
             'status_refused' => Signalement::STATUS_REFUSED,
         ];
 
+        $territories = [];
         if (null !== $territory) {
```

Comment:
En SA, la liste des signalement ne se charge pas en étant filtrée sur http://localhost:8080/bo/signalements/?sansSuiviPeriode=30&sortBy=lastSuiviAt&direction=DESC&territoire=319

Erreur : 

App\Repository\SuiviRepository::findSignalementNoSuiviSince(): Argument #2 ($territory) must be of type ?App\Entity\Territory, int given, called in /app/src/Service/Signalement/SearchFilter.php on line 468

Reply:
j'ai ce même pb en RT du 13, quand j'acccède à la liste via le widget "Sans suivi" du dashboard

Reply:
idem en utilisateur multi-territoire si je choisis un territoire, mais pas si je n'en chcoisis pas